draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1567,9 +1567,10 @@ After the validation, the Verifier will have the following data for further proc
 
 ## Example 4a - SD-JWT-based Verifiable Credentials (SD-JWT VC)
 
-In this example, the artifacts defined in this specification are used to represent
-SD-JWT-based Verifiable Credentials (SD-JWT VC) as defined in [@I-D.ietf-oauth-sd-jwt-vc].
-Person Identification Data (PID) defined in [@EUDIW.ARF] is used.
+This example shows how the artifacts defined in this specification
+could hypothetically be used to express an
+SD-JWT-based Verifiable Credential [@I-D.ietf-oauth-sd-jwt-vc] with
+Person Identification Data (PID) defined in [@EUDIW.ARF].
```

Comment:
```suggestion
This is a non-normative example how the artifacts defined in this specification
could hypothetically be used to express an
SD-JWT-based Verifiable Credential [@I-D.ietf-oauth-sd-jwt-vc] with
Person Identification Data (PID) defined in [@EUDIW.ARF].
```

Reply:
```suggestion
This example shows how the artifacts defined in this specification could be used to represent
SD-JWT-based Verifiable Credentials (SD-JWT VC) as defined in [@I-D.ietf-oauth-sd-jwt-vc].
Person Identification Data (PID) defined in [@EUDIW.ARF] is used as a sample payload.
```

Reply:
ok, so I think editors discussed to merge #407 first, and not cover in this PR?

Reply:
#407 was merged and addresses this part